SUMMARY.md

```diff
@@ -14,13 +14,16 @@
   - [Alert Handler](./hw/top_earlgrey/ip_autogen/alert_handler/README.md)
     - [Design Verification](./hw/top_earlgrey/ip_autogen/alert_handler/dv/README.md)
     - [Interface and Registers](./hw/top_earlgrey/ip_autogen/alert_handler/data/alert_handler.hjson)
+    - [Device Interface Functions](./sw/device/lib/dif/dif_alert_handler.h)
```

Comment:
For this commit message and others that start with a verb, I think the convention is to capitalize the first letter after the tag.